salt/apps/ocw/symlinks_origin.sls

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
It's probably worth trying to move the files that we're serving to a more sane directory structure (e.g. /var/www/ocw) and then symlinking that to the weird tomcat directory that we're stuck with so that it's easier to factor that out to the sane target down the road.

Reply:
If I understand you correctly, that's what's being done [here](https://github.com/mitodl/salt-ops/blob/ab25652e98005bed2072fdd522fc0b5cf95e4f88/salt/apps/ocw/sync_repo.sls#L46-L59).

These symlinks are only being created to allow this script to function: https://github.com/mitocw/ocwcms/blob/add-missing-files/publishing/sitemap.sh
That script runs on the origin server, and is invoked by https://github.com/mitocw/ocwcms/blob/add-missing-files/publishing/run_aka_scripts.sh on the cms2 engines server. We just want to preserve the crazy filesystem locations for the time being in case files get copied back to the legacy servers.